spec/views/users/_follower_modal.html.erb_spec.rb

```diff
@@ -18,36 +18,55 @@
 
 describe 'users/_follower_modal.html.erb', :type => :view do
 
-  let(:frank) { FactoryGirl.create(:user, display_name: "Frank") }
   before do
-    assign :user, frank
     assign :followers, []
   end
 
-  describe "when current user has no followers" do
-
+  context "when logged in " do
+    let(:frank) { FactoryGirl.create(:user, display_name: "Frank") }
     before do
-      allow(controller).to receive(:current_user).and_return(frank)
+      assign :user, frank
     end
 
-    it "should indicate the lack of followers for you" do
-      render
-      page = Capybara::Node::Simple.new(rendered)
-      expect(page).to have_text "No one is following you." 
+    describe "when current user has no followers" do
+
+      before do
+        allow(controller).to receive(:current_user).and_return(frank)
+      end
+
+      it "should indicate the lack of followers for you" do
```

Comment:
Hi, no, definitely not. The way GitHub formatted this fooled me -- I didn't see that this was just a spacing change.  Nvm!